asmcli/cloudbuild-main.yaml

```diff
@@ -6,17 +6,22 @@ steps:
   - name: 'gcr.io/cloud-builders/bazel'
     entrypoint: 'bazel'
     args: ['build', 'merge']
```

Comment:
We should put this after the image build.

Reply:
Done